types/index.d.ts

```diff
@@ -672,17 +680,23 @@ declare namespace math {
      */
     compile(exprs: MathExpression[]): EvalFunction[]
 
+    // TODO properly type this
```

Comment:
@mattvague does this TODO still need to be addressed?

Reply:
This was a reminder for another PR, made a task here: https://github.com/josdejong/mathjs/discussions/2565